test/parallel/test-node-run.js

```diff
@@ -223,15 +224,19 @@ describe('node --run [command]', () => {
     assert.strictEqual(child.code, 1);
   });
 
-  it('runs script in a custom working directory using --run-in-directory', async () => {
+  it('runs script in a custom working directory using --run-dir', async () => {
```

Comment:
You need to add a new test that passes a non-directory path to --run-dir and you should validate the output of it.

Reply:
thanks